openlibrary/i18n/__init__.py

```diff
@@ -13,15 +13,12 @@
 root = os.path.dirname(__file__)
 
 def _compile_translation(po, mo):
-    try:
-        catalog = read_po(open(po))
-
-        f = open(mo, 'wb')
-        write_mo(f, catalog)
-        f.close()
-        print('compiled', po, file=web.debug)
-    except:
-        print('failed to compile', po, file=web.debug)
+    catalog = read_po(open(po))
+
+    f = open(mo, 'wb')
+    write_mo(f, catalog)
+    f.close()
+    print('compiled', po, file=web.debug)
```

Comment:
We could remove this to make the logs less chatty. I'm not sure it adds much value.

Reply:
For now I find it kind of useful (and rewarding :P) To see the languages there one by one; once we have 100s of languages we should remove it though 😉 (Won't fix). It's also consistent with how the other tests work (i.e. less, pytest).